compiler/testData/diagnostics/tests/varargs/varargInSetter.fir.kt

```diff
@@ -1,5 +1,5 @@
 class My {
-    var x: String = ""
+    var x: <!WRONG_SETTER_PARAMETER_TYPE!>String<!> = ""
```

Comment:
This report seems not right either: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/varargs/varargInSetter.kt#L2